assets/css/_picker_container.scss

```diff
@@ -1,39 +1,58 @@
 .m-picker-container {
-  background-color: $color-bg-medium;
+  background-color: #ecedee;
```

Comment:
**suggestion (non-blocking):** This value is used in several places. It might be worthwhile turning it into a variable.